.gitignore

```
@@ -24,3 +24,6 @@ dist-ssr
 *.sln
 *.sw?
 coverage
+
+tsconfig.app.tsbuildinfo
+tsconfig.node.tsbuildinfo
```

you can replace them with a single, cleaner line
```suggestion
*.tsbuildinfo
```